src/main/resources/db/migration/V001__Init.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE letter (
+  id BIGSERIAL NOT NULL PRIMARY KEY,
+  message_id UUID NOT NULL,
```

Comment:
message id is md5 hash. just varchar I guess...

Reply:
probably rename table now too